Contributing.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to this project:
+
+We appreciate your interest in supporting [AI-Powered-News-Article-Classifier-with-History]! . Your contributions are highly valued and can take various forms. This guide outlines the different ways you can contribute to the project.
+
+# To contribute to this repository:
+
+The following are some ways you can contribute to this project:
+
+## 1.Report Bugs:
+
+If you encounter any bugs or problems while working on the project, kindly let us know. Describe the issue in full, including how to reproduce it and any error messages you run into, in an issue on the project's GitHub repository.
+
+## 2.Request Features:
+
+If you have suggestions for new features or improvements, please share them with us. Declare the feature you want to be included as an issue on the project's GitHub repository. Your suggestions aid us in setting priorities and guiding the project's progress.
+
+## 3.Changes to the Code:
+
+If you're interested in making changes to the project's code, please do so. In order to submit your code changes, follow these steps:
+
+### Step 1: Fork this repository
+
+●Click the "Fork" button at the top right corner of this repository's page.
+
+![Alt text](images/fork.png)
```

Comment:
can you please remove Alt text
and add `<img src="YOUR_IMAGE_URL"  width="{Between_300 - 500_px}" >`

Reply:
If you give me the edit access I can show you
@Kanchan253

Reply:
great i'll made some changes and Merge the PR By Monday

Reply:
Ok sure

Reply:
Thanks you so much for your contribution. I'll show your profile in in repo readme for sure 
Wish you a good luck

Reply:
Thank you!!